pkg/oci/containerize_test.go

```diff
@@ -0,0 +1,43 @@
+package oci
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func Test_validateLink(t *testing.T) {
+	root := "testdata/test-links"
+
+	tests := []struct {
+		path  string // path of the file within test project root
+		valid bool   // If it should be considered valid
+		name  string // descriptive name of the test
+	}{
+		{"a.txt", true, "do not evaluate regular files"},
+		{"a.lnk", true, "do not evaluate directories"},
```

Comment:
Shouldn't be there a directory name, not link name?

Reply:
Sorry that was just a typo while refactoring.